config.py.docker

```diff
@@ -0,0 +1,149 @@
+#
+# config.py - Global configuration constants and runtime info
+#
+
+import logging, time
+
+# Config - defines
+
+
+class Config:
+    #####
+    # Part 1: Tango constants for developers
+    #
+    # These allow developers to run test versions of Tango on the same
+    # server as the production Tango
+
+    # Unique prefix that defines VM name space for this Tango
+    # version. When working in development, this prefix should be your
+    # unique identifier. The "prod" prefix is reserved for production
+    PREFIX = "prod"
```

Comment:
I think this file looks very similar to Tango/config.template.py which already exists, except that under one-click set up environment `PREFIX` and  `DOCKER_VOLUME_PATH` are changed. Maybe another approach is to set an env variable in Dockerfile, so we can do something like: 
```
PREFIX = "prod" if os.getenv("ONE_CLICK") else "local"
DOCKER_VOLUME_PATH = "/opt/TangoService/Tango/volumes/" if os.getenv("ONE_CLICK") else ""
```
so that we can avoid having two very similar files in the repo.

Reply:
Yeah, this is kind of a tough problem and it's only going to get worse with Autolab config files which you'll see soon. Agree with you that ideally we should just have a single source of truth which makes maintenance easier in the future. Let's discuss about this later and see if there's a better way to solve this problem generally.